src/crewai/cli/utils.py

```diff
@@ -300,12 +299,45 @@ def get_crew(crew_path: str = "crew.py", require: bool = False) -> Crew | None:
         if require:
             console.print("No valid Crew instance found in crew.py", style="bold red")
             raise SystemExit
-        return None
 
     except Exception as e:
         if require:
             console.print(
                 f"Unexpected error while loading crew: {str(e)}", style="bold red"
             )
             raise SystemExit
-        return None
+    return crew_instances
+
+
+def get_crew_instance(module_attr) -> tuple[bool, Crew | None]:
```

Comment:
do we need to return a bool as well? If Crew is returned then we can assume its True?

Reply:
nice call